packages/ppx_map/ppx_map.0.1.0/opam

```diff
@@ -0,0 +1,38 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "A PPX rewriter to simplify the declaration of maps"
+description:
+  "This package allows to declare maps in an expressive way using [key => value] expressions"
+maintainer: ["Benjamin Somers <benjamin.somers@imt-atlantique.fr>"]
+authors: ["Benjamin Somers <benjamin.somers@imt-atlantique.fr>"]
+license: "MIT"
+homepage: "https://github.com/bensmrs/ppx_map"
+bug-reports: "https://github.com/bensmrs/ppx_map/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "alcotest" {>= "1.6.0"}
```

Comment:
This dependency seems to only be needed for the tests
```suggestion
  "alcotest" {with-test & >= "1.6.0"}
```

Reply:
Well… I didn’t know about `with-test`, still a lot to learn!